app/admin/notifications.rb

```diff
@@ -0,0 +1,25 @@
+ActiveAdmin.register Notification do
+  permit_params :user, :details, :viewed
+
+  index sortable: :viewed_asc do
+    column :id
+    column 'Sender', sortable: :user_id do |notification|
```

Comment:
If the `user_id` is the person who received the email, then "Receiver" would be more clear than "Sender".

Reply:
@rogergraves `user_id `is the person who send (create) email.